src/mcnp/mcnp_funcs.cpp

```diff
@@ -123,15 +123,16 @@ void dagmcwritefacets_(char* ffile, int* flen) { // facet file
   // out to the fcad file
   if (workflow_data->material_library.size() != 0) {
     // get the mat lib
-    std::map<std::string, pyne::Material> mat_lib = workflow_data->material_library;
-    std::map<std::string, pyne::Material> ::iterator it;
+    pyne::MaterialLibrary mat_lib = workflow_data->material_library;
```

Comment:
I know the original code did this, but does it need to be a copy?  Could this just be a reference?